### DISCUSSION TITLE
Disable Piped Proxy includes everything except Piped feed and playlists

### DISCUSSION THREAD

**Discussion by trymeouteh** [libre-tube/LibreTube#3423] [Ideas]
Please have the "Disable Piped Proxy" disable the piped proxy for everything except for your Piped feed and Piped playlists. This will mean that it will grab all data from youtube.com including video description, comments, search results, homepage, likes, etc. This way if your Piped instance is down, only your feed and playlists are unavailable.

**Comment by Bnyro** [comment]
That's out of scope for the project!
